content/applications/hr/appraisals.rst

```diff
@@ -0,0 +1,479 @@
+and how they can improve.
+
+Configuration
+=============
+
+The configuration menu is where the appraisal feedback template is edited, the frequency is set,
+the evaluation scale is managed, and where the data for the 360 feedback is stored.
+
+Settings
+--------
+
+To access the :guilabel:`Settings` menu, navigate to :menuselection:`Appraisals application -->
+Configuration --> Settings`.
+
+Feedback templates
+~~~~~~~~~~~~~~~~~~
+
+Feedback templates provide the outline of the form used during an employee appraisal. Any edits made
+to this template are reflected in the appraisals sent to employees.
+
+There are two default templates pre-configured for both employee feedback and manager feedback, each
+with several sections containing questions and brief explanations on how to respond to the
+questions. The :guilabel:`Employee Feedback Template` has the following sections: :guilabel:`My
+work`, :guilabel:`My future`, and :guilabel:`My feelings`. The :guilabel:`Manager Feedback Template`
+has the following sections: :guilabel:`Feedback`, :guilabel:`Evaluation`, and
+:guilabel:`Improvements`.
+
+Make any desired changes to the feedback templates by making changes directly in each template.
+
+Appraisals
+~~~~~~~~~~
+
+The :guilabel:`Appraisal` section of the settings menu deals with the frequency that appraisals are
+performed, and if other feedback is available to request.
+
+.. image:: appraisals/appraisals-setting.png
+   :align: center
+   :alt: The appraisals sections with the timeline filled in and 360 feedback enabled.
+
+.. _appraisals/appraisal-plan:
+
+Appraisals plans
+****************
+
+Appraisals are pre-configured to be automatically generated six months after an employee is hired,
+then another appraisal six months after that. After those initial two appraisals that first year,
+appraisals are conducted once a year (every twelve months).
+
+To modify the schedule, change the number of months in the blank fields under the
+:guilabel:`Appraisal Plans` section.
+
+.. important::
+   If the :guilabel:`Appraisals Plans` section is modified, all empty :guilabel:`Next Appraisal
+   Dates` are modified for all employees.
+
+360 feedback
+************
+
+Enable the :guilabel:`360 Feedback` option to allow managers to request feedback from other
+employees using a different survey form, at any time, independent of the appraisal schedule.
+Typically, managers ask for feedback from other people who work with an employee they manage. This
+includes the employee's various manager, peers, and direct reports.
+
+To view the :guilabel:`360 Degree Feedback` survey, click the :guilabel:`→ Internal link` icon after
+the :guilabel:`Default Template` field. The :guilabel:`360 Feedback` survey loads. Make any desired
+changes to the survey here. For more information on how to edit a survey, refer to the
+:doc:`../marketing/surveys/create` document.
+
+.. note::
```

Comment:
I try to not have admonition blocks at the top, directly beneath a section- but that's just a personal preference. I will change this to an "important" since this is important and not just a casual note.